account-decoder/src/parse_account_data.rs

```diff
@@ -111,8 +111,8 @@ mod test {
 
     #[test]
     fn test_parse_account_data() {
-        let account_pubkey = Pubkey::new_rand();
-        let other_program = Pubkey::new_rand();
+        let account_pubkey = solana_sdk::pubkey::new_rand();
```

Comment:
Stating `solana_sdk` everytime is pretty verbose, we would probably want:

```rust
use solana_sdk::pubkey;

...

let account_pubkey = pubkey::new_rand();
```

Reply:
that was just an artifact of mass search/replace

Reply:
I'm going to leave this as is, it can be cleaned up later.   It'll be a painful manual edit to over a hundred files